jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseServiceImplTest.java

```diff
@@ -876,6 +876,65 @@ public void testCaseWithComments() {
             }
         }
     }
+    
+    @Test
+    public void testCaseWithCommentsPagination() {
```

Comment:
@tsurdilo saw your last comment and good point.  i have something working locally now.. what i did was refactor the method signature to pass in a boolean parameter for pagination.  if this is set to false, it will return all comments.  but not sure if that's the best solution, or if it would be best to remove the default count on the QueryContext.. @mswiderski @cristianonicolai